Add EN Blog: Kubernetes High Availability Essential Practices Simply Explained

Felixnoo opened the pull request:
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

I uploaded [this blog](https://hackmd.io/-XRwruzJQ_KT_CLXsh2cuA?view). Please take a look, thanks. -> [Preview link](https://deploy-preview-2299--kubesphere-v3-1.netlify.app/blogs/k8s-ha-practices/).

/assign @JiannaLiu @FeynmanZhou @Bettygogo2021

FeynmanZhou commented:
The original version is supposed to be simplified and refined. For example, we don't need to include the KubeSphere installation since the topic is only for illustrating the K8s HA. 

I also modified this blog from my side before, you can find it on [Google Doc](https://docs.google.com/document/d/1pSLMZLRsmx0ynemWhR453vmkg314ZXevhcip7bwk5oA/edit#).

Felixnoo commented:
> The original version is supposed to be simplified and refined. For example, we don't need to include the KubeSphere installation since the topic is only for illustrating the K8s HA.
> 
> I also modified this blog from my side before, you can find it on [Google Doc](https://docs.google.com/document/d/1pSLMZLRsmx0ynemWhR453vmkg314ZXevhcip7bwk5oA/edit#).

Thanks for the suggestion. Updated.

FeynmanZhou commented:
/lgtm
/approve
